docs/src/further_topics/ugrid/operations.rst

```diff
@@ -0,0 +1,953 @@
+            long_name: 'my_mesh'
+            var_name: 'my_mesh'
+
+Plotting
+--------
+.. |tagline: plotting| replace:: |different| - plot with GeoVista
+
+.. rubric:: |tagline: plotting|
+
+The Cartopy-Matplotlib combination is not optimised for displaying the high
+number of irregular shapes associated with meshes. Thankfully mesh
+visualisation is already popular in many other fields (e.g. CGI, gaming,
+SEM microscopy), so there is a wealth of tooling available, which
+:ref:`ugrid geovista` harnesses for cartographic plotting.
+
+GeoVista's default behaviour is to convert lat-lon information into full XYZ
+coordinates so the data is visualised on the surface of a 3D globe. The plots
+are interactive by default, so it's easy to explore the data in detail.
+
+2D projections have also been demonstrated in proofs of concept, and will
+be added to API in the near future.
+
+This first example uses GeoVista to plot the ``face_cube`` that we created
+earlier:
+
+.. dropdown:: :opticon:`code`
+
+    .. code-block:: python
+
+        >>> from geovista import GeoPlotter, Transform
+        >>> from geovista.common import to_xyz
+
+
+        # We'll re-use this to plot some real global data later.
+        >>> def cube_faces_to_polydata(cube):
+        ...     lons, lats = cube.mesh.node_coords
+        ...     face_node = cube.mesh.face_node_connectivity
+        ...     indices = face_node.indices_by_location()
+        ...
+        ...     mesh = Transform.from_unstructured(
+        ...         lons.points,
+        ...         lats.points,
+        ...         indices,
+        ...         data=cube.data,
+        ...         name=f"{cube.name()} / {cube.units}",
+        ...         start_index=face_node.start_index,
+        ...     )
+        ...     return mesh
+
+        >>> print(face_cube)
+        face_data / (K)                     (-- : 2; height: 3)
+            Dimension coordinates:
+                height                          -          x
+            Mesh coordinates:
+                latitude                        x          -
+                longitude                       x          -
+            Attributes:
+                Conventions                 CF-1.7
+
+        # Convert our mesh+data to a PolyData object.
+        # Just plotting a single height level.
+        >>> face_polydata = cube_faces_to_polydata(face_cube[:, 0])
+        >>> print(face_polydata)
+        PolyData (0x7ff4861ff4c0)
+          N Cells:	2
+          N Points:	5
+          X Bounds:	9.903e-01, 1.000e+00
+          Y Bounds:	0.000e+00, 1.392e-01
+          Z Bounds:	6.123e-17, 5.234e-02
+          N Arrays:	2
+
+        # Create the GeoVista plotter and add our mesh+data to it.
+        >>> my_plotter = GeoPlotter()
+        >>> my_plotter.add_coastlines(color="black")
+        >>> my_plotter.add_base_layer(color="grey")
+        >>> my_plotter.add_mesh(face_polydata)
+
+        # Centre the camera on the data.
```

Comment:
These lines here about the camera are just touching over the boilerplate tolerance level, for me.
So I hope this isn't done for showiness, but rather is essential to ensure the resulting plot is useful out of the box ??
If there was any way of reducing the "extra stuff" a bit here, I'd really like that

Reply:
I see that the next example in line is much cleaner-looking.

Reply:
Yes, that's why it's there.

Are you basically saying you'd prefer it if we set the camera position on a single line, using raw coordinates?

Reply:
I'm not really capable of making any concrete proposal here  -- I'm just a little disappointed that the very first example makes it look so complicated

Reply:
Maybe you can just pull out one specific point + centre it on that ??

Reply:
... I don't really understand the whole "region" thing here though

Reply:
I've looked at this some more and the alternative would be:

```python
my_plotter.camera_position = [3, 0.183, 0.079]
```

Which, while shorter, is unhelpful for the user when replicating with their own data. I expect camera centre-ing to be a very common operation with any non-global data, and until GeoVista includes conveniences for it I feel it's important to include lines that show how to generically achieve this, so I'm not going to action this one.